demos/notifications/cayman.css

```diff
@@ -0,0 +1,241 @@
+* {
```

Comment:
Whether we should use some external css theme for demo page at this stage is debatable. Either ways the file is external and should go in external directory @arschmitz thoughts?